CWP/papers/HSF-CWP-2017-01_roadmap/latex/cwp-main.tex

```diff
@@ -2709,7 +2708,9 @@ \subsubsection*{Research and Development Roadmap}
 visualisation needs in production, potentially taking advantage of new
 data access services.
 
-The workshop that was held as part of the CWP process was felt to be
+The workshop that was held as part of the CWP process
+(\emph{HSF Visualization Workshop}, see Appendix \ref{appendix-a---list-of-workshops})
```

Comment:
Guess this is something you want to finalise?

Reply:
There are 2 workshops listed at the bottom of https://hepsoftwarefoundation.org/workinggroups/visualization.html. Anything else? One of the 2 is in fact already in Appendix A.

Reply:
I swithered over making an actual citation to the Indico agenda, but then as the workshop was already listed in the appendix I thought that a reference to the appendix was enough. Maybe we just say workshop*s*?

Reply:
That's fine by me, yes.